src/main.c

```diff
@@ -2583,6 +2586,27 @@ void reset_model() {
     g->time_changed = 1;
 }
 
+/**This function plays audio using MPG321 according to the argument passed in.*/
+int play_audio_source(char *type_of_music)
```

Comment:
You should switch your if/else statements into a switch statement to make it easier to read.

Reply:
For now I will keep the if/else statements. If it becomes overbearingly unreadable in the future only then I will consider such a change.